pom.xml

```diff
@@ -405,6 +405,13 @@
     </build>
 
     <repositories>
+        <!-- Maven repository -->
+<!--        <repository>-->
```

Comment:
I am guessing this wasn't intended to be part of the pull request? Of so, please remove it in a subsequent commit.